client/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import 'bootstrap/dist/css/bootstrap.css';
+
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
```

Comment:
You should not add components in the `index.js` file , instead this you can add components in `app.js`